apps/epic-web/src/inngest/functions/sanity/product/sanity-product-created.ts

```diff
@@ -82,6 +83,114 @@ export const sanityProductCreated = inngest.createFunction(
       })
     })
 
+    type SplitInfo = {
+      type: string
```

Comment:
type here is `'owner' | 'skill' | string`

could use a [discriminated union](https://www.totaltypescript.com/discriminated-unions-are-a-devs-best-friend) so that the `userId would only be there if it was type `owner`